godu.go

```diff
@@ -43,6 +46,23 @@ func main() {
 	printMarkedFiles(lastState)
 }
 
+func reportProgress(progress <-chan int) {
+	objs := 0
+	ticker := time.NewTicker(time.Second * 2)
+	go func() {
+		for i := range progress {
+			objs += i
+		}
+		ticker.Stop()
+		log.Println("Scanning done")
+	}()
+	go func() {
+		for range ticker.C {
+			log.Printf("Scanning.. Already found %d objects\n", objs)
```

Comment:
That's a good point. We are now using folder/tree/file, often interchangeably. I think we should come up with naming convention going forward. I was thinking about calling everything `file`, but that might be  more confusing. Maybe we can say that `folder` is a special case of `file` and stop using `tree` completely? What's your opinion?